lib/xboxrt/libc_extensions/string_ext_.h

```diff
@@ -33,3 +33,88 @@ __attribute__((deprecated)) static int stricmp (const char *s1, const char *s2)
 #ifdef __cplusplus
 }
 #endif
+
+
+#ifdef __cplusplus
+// libc++ expects MSVCRT to provide these overloads
+
+#ifndef _NXDK_PREFERRED_OVERLOAD
+#define _NXDK_PREFERRED_OVERLOAD __attribute__ ((__enable_if__(true, "")))
```

Comment:
Do we use this `_NXDK` stuff elsewhere too?

I don't see this as being specific to nxdk.

I feel like this should be using a `_PDCLIB` prefix (or whatever upstream PDCLIB would use) and that might even be a good candidate to upstream in the future, or to keep it neutral for re-use in other projects.
If there are collisions, maybe `_PDCLIB_EXT` or similar.

Reply:
Not yet afaik.
I'd say these are specific to nxdk, as they're meant to provide the overloads that libc++ expects from the MSVC runtime library.

I don't think we should use the `PDCLib` namespace here - it's not a direct part of PDCLib, and I'd like to avoid potential naming conflicts.

Reply:
I didn't have time to respond to this before this got merged. But I disagree with this.
These functions aren't nxdk specific at all, and they should have been prefixed differently.